packages/account-v2/src/containers/Onfido/OnfidoView.classnames.ts

```diff
@@ -9,7 +9,7 @@ export const onfidoCustomClassVariant = cva('onfido-custom-container', {
 });
 
 export const onfidoInfoMessageVariant = cva(
-    'absolute p-400 top-200 min-h-1700 text-center z-[1] w-full [transition:transform_0.35s_linear_4.65s] origin-top',
+    'absolute p-8 top-4 min-h-34 text-center z-[1] w-full [transition:transform_0.35s_linear_4.65s] origin-top',
```

Comment:
maybe we need to look into this position property as well where it is being used